tripy/tripy/backend/compiler_api.py

```diff
@@ -31,6 +31,130 @@
 from tripy.common.shape_bounds import ShapeBounds
 from tripy.frontend import Tensor, Trace
 from tripy.utils import json as json_utils
+from tripy.common.device import device as Device
+
+
+# Create default stream.
+@export.public_api(document_under="compiler")
+class default_stream:
```

Comment:
This might actually be better as a function. Right now it's a class whose constructor returns an object of a different type than an instance of the class.

Reply:
Moved to a function